frontend/src/components/shortcut/Shortcut.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang=ts>
+    import { createEventDispatcher } from 'svelte';
+
+    // Label attached to the input text. Desciption of the shortcut
+    export let shortcut_label: string = "Not defined";
+    // Key to be displayed within the input text
+    export let shortcut_key: string;
+
+    const dispatch: Function = createEventDispatcher();
+
+    // Dispatch an event when the value of the input has been changed. Useful for rebinding
+    const change_event = (event: Event) => {
+        dispatch('change', event)
```

Comment:
event seems to be unused atm

Reply:
Yes, it is made with the possibility to rebind in mind. But if you rather like, I can remove the code related to rebinding